content/en/docs/ibm/deploy/authentication.md

```diff
@@ -110,38 +110,165 @@ column):
     rm istio-ingressgateway-certs.yaml
     ```
 
-7. Update the gateway `kubeflow-gateway` to expose port `443`. Create a resource 
-file `kubeflow-gateway.yaml` as follows by replacing `<hostname>` with the value 
+7. Update the gateway `kubeflow-gateway` to expose port `443`. Create a resource
+file `kubeflow-gateway.yaml` as follows by replacing `<hostname>` with the value
 of the column `Hostname` in step 4:
 
     ```YAML
     apiVersion: networking.istio.io/v1alpha3
     kind: Gateway
     metadata:
-    name: kubeflow-gateway
-    namespace: kubeflow
+      name: kubeflow-gateway
+      namespace: kubeflow
     spec:
       selector:
         istio: ingressgateway
-    servers:
-    - hosts:
-      - '<hostname>'
-      port:
-        name: https
-        number: 443
-        protocol: HTTPS
-      tls:
-        mode: SIMPLE
-        privateKey: /etc/istio/ingressgateway-certs/tls.key
-        serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
-    ```
-
-8. Verify that the traffic is routed via HTTPS by using the value of 
-above-mentioned `Hostname` in your browser. It should redirect traffic from an 
+      servers:
+      - hosts:
+        - '<hostname>'
+        port:
+          name: https
+          number: 443
+          protocol: HTTPS
+        tls:
+          mode: SIMPLE
+          privateKey: /etc/istio/ingressgateway-certs/tls.key
+          serverCertificate: /etc/istio/ingressgateway-certs/tls.crt
+    ```
+
+8. Verify that the traffic is routed via HTTPS by using the value of
+above-mentioned `Hostname` in your browser. It should redirect traffic from an
 HTTP address to HTTPS address automatically.
 
-**Note**: The certificates for the NLB DNS host secret expire every **90** days. 
+**Note**: The certificates for the NLB DNS host secret expire every **90** days.
 The secret in the `default` namespace is automatically renewed by IBM Cloud
-Kubernetes Service 37 days before it expires. After this secret is updated, you 
-must manually copy it to the `istio-ingressgateway-certs` secret by repeating 
-commands in step 5 and 6.
\ No newline at end of file
+Kubernetes Service 37 days before it expires. After this secret is updated, you
+must manually copy it to the `istio-ingressgateway-certs` secret by repeating
+commands in step 5 and 6.
+
+## Optional - KFServing configuration
+
+With this HTTPS setup, additional changes are needed to get KFServing to work.
```

Comment:
Suggestion:

- Change to active voice from passive (to help the users by directly addressing):


```suggestion
With this HTTPS setup, you need to make additional changes to get KFServing to work.
```